src/location-search/LocationSearch.tsx

```diff
@@ -178,52 +178,53 @@ const LocationSearch: React.FC<Props> = ({
             contentContainerStyle={styles.contentBlock}
           />
         </View>
+      </View>
 
-        {error && (
-          <View style={styles.contentBlock}>
-            <MessageBox
-              type="warning"
-              message={errorMessage}
-              containerStyle={{marginBottom: 12}}
+      {error && (
+        <View style={styles.contentBlock}>
+          <MessageBox
+            type="warning"
+            message={errorMessage}
+            containerStyle={{marginBottom: 12}}
```

Comment:
This little guy has managed to escape every theming effort so far... Wonder if it will survive this time. :thinking:

Reply:
Hirr 😄  I'll fix it. The added indentation (due to wrapped element) will cause this to show my name in git blame 🚀

Reply:
Fixed